### PR TITLE
fix(#1698): dropdown width % should respect width container

### PR DESCRIPTION
**Pull request by vanessatran-ddi**
# Before (the change)
Resize browser and width of the dropdown isn't updated. 
[Bug 1698](https://github.com/GovAlta/ui-components/issues/1698)
# After (the change)
https://jam.dev/c/fc041636-9167-40c9-82ae-8fb85a7eaad9

## Make sure that you've checked the boxes below before you submit the PR

- [x] I have read and followed the [setup steps](https://goa-dio.atlassian.net/wiki/spaces/DS/pages/3086385489/Contribution+-+web+content#%5BinlineExtension%5DCode-contributions)
- [x] I have created necessary unit tests
- [x] I have tested the functionality in both React and Angular.

## Steps needed to test

Code snippet: 
```
 <div style={{width: "80%", border: "2px solid red"}}>
        <GoAFormItem label="Input" requirement="required">
          <GoAInput name="input" placeholder="This is input" width="100%" onChange={onChange}></GoAInput>
          {/*<GoADropdown name="item"  onChange={onChange} width="100%">*/}
          {/*  <GoADropdownItem value="red" label="Red" />*/}
          {/*  <GoADropdownItem value="green" label="Green" />*/}
          {/*  <GoADropdownItem value="blue" label="Blue" />*/}
          {/*</GoADropdown>*/}
        </GoAFormItem>
        <GoAFormItem label="Item" requirement="required">
        <GoADropdown name="item"  onChange={onChange} placeholder="This is a dropdown width 100%" width="100%">
          <GoADropdownItem value="red" label="Red" />
          <GoADropdownItem value="green" label="Green" />
          <GoADropdownItem value="blue" label="Blue" />
        </GoADropdown>
        </GoAFormItem>

        <GoAFormItem label="Item" requirement="required">
          <GoADropdown name="item"  onChange={onChange} placeholder="Normal dropdown">
            <GoADropdownItem value="red" label="Red" />
            <GoADropdownItem value="green" label="Green" />
            <GoADropdownItem value="blue" label="Blue" />
          </GoADropdown>
        </GoAFormItem>
      </div>
```

### CODE REVIEW

**Comment by lizhuomeng71** [comment]
when the width of input is changed, it double applied twice

For example, this is 50%

image

image

File: libs/web-components/src/components/dropdown/Dropdown.svelte [hunk 1]
```diff
@@ -501,7 +490,7 @@
 </script>
 
 <!-- Template -->
-<div bind:this={_wrapperEl}>
+<div>
```
Comment (chrisolsen) [inline libs/web-components/src/components/dropdown/Dropdown.svelte:493]: This div can be removed now

**Comment by lizhuomeng71** [comment]
issue is fixed, verified on react and angular

**Review by chrisolsen — APPROVED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**